chemicaltoolbox/xchem-deep/xchem_deep_score.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="xchem_pose_scoring" name="XChem pose scoring" version="0.1.0">
+    <description>- perform pose scoring using deep learning</description>
+
+    <requirements>
+        <!--requirement type="package" version="3.0.0">openbabel</requirement-->
+        <!--requirement type="package" version="3.7">python</requirement-->
+        <!-- many other requirements are needed -->
+        <container type="docker">informaticsmatters/deep-app-ubuntu-1604:latest</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s '$ligands' ligands.sdf &&
+    ln -s '$receptor' receptor.pdb &&
+    python '$__tool_directory__/xchem_deep_score.py' -i ligands.sdf -r receptor.pdb --mock &&
```

Comment:
we can make the --mock parameter a real Galaxy param with type="hidden" ... with this we can set this to true for the test case ...

Reply:
https://github.com/tdudgeon/galaxytools/commit/4427a2cffd63113138773495a3f796e67ce2dcf2
@bgruening Can you check if this is done correctly?